website/docs/serialization/features.mdx

```diff
@@ -0,0 +1,14 @@
+---
+id: features-molecule
+title: Features
+---
+
+import Tooltip from "@components/Tooltip";
+
+# Features
+
+The `std` feature, contained in `default`, is enabled by default and comprises:
```

Comment:
Molecule is developed in Rust language. The `default` feature in the Rust library is enabled by default, including `std` feature and more: